docs/platformdev_blackpill_f411.md

```diff
@@ -0,0 +1,42 @@
+# WeAct Blackpill (STM32F411)
+
+The WeAct Blackpill is a popular choice for handwired boards, as it offers a powerful micro controller, USB Type C, a good number of pins to use, and a large amount of firmware space.  All for a ~$6 USD price tag. 
+
+* [Official WeAct Store (AliExpress)](https://fr.aliexpress.com/item/1005001456186625.html)
```

Comment:
Currently links to the AT32 stuff, maybe remove it?

Reply:
Also if you really want to link to AliExpress, please use `www.aliexpress.com` instead of a country-specific domain (following a country-specific link on AliExpress often switches the site language, which is annoying).